addons/godot_piskel_importer/piskel_importer.gd

```diff
@@ -1,50 +1,55 @@
-tool
+@tool
 extends EditorImportPlugin
 
-func get_importer_name():
+func _get_importer_name():
 	return "com.technohacker.piskel"
 
-func get_visible_name():
+func _get_visible_name():
 	return "Piskel Project"
 
-func get_recognized_extensions():
+func _get_recognized_extensions():
 	return ["piskel"]
 
-# We save directly to stex because ImageTexture doesn't work for some reason
-func get_save_extension():
-	return "stex"
+func _get_save_extension():
+	return "tres"
 
-func get_resource_type():
+func _get_resource_type():
 	return "StreamTexture"
```

Comment:
Should this be ImageTexture instead?

Reply:
Yes, you are right. StreamTexutre no longer exists in Godot 4.0

Reply:
Alright I'll commit that now :)
```suggestion
	return "ImageTexture"
```